electron_app/src/electron-main.js

```diff
@@ -42,7 +42,9 @@ const Store = require('electron-store');
 // migrating to mitigate any risk of it being used maliciously.
 let migratingOrigin = false;
 
-if (argv['profile']) {
+if (argv['profiledir']) {
+    app.setPath('userData', argv['profiledir']);
```

Comment:
Since the flag name means two words, I would recommend using `profile-dir` instead. This match existing multi-word flags like `no-update`.

Reply:
Fixed!